src/dna_string.rs

```diff
@@ -50,7 +50,7 @@ const MASK: u64 = 0x3;
 /// A container for sequence of DNA bases.
 #[derive(Ord, PartialOrd, Clone, PartialEq, Eq, Hash, Serialize, Deserialize)]
 pub struct DnaString {
-    storage: Vec<u64>,
+    pub storage: Vec<u64>,
```

Comment:
We definitely don't want this field to be public.  Make `ndiffs` into a static method inside the `impl DnaString {}` block and you'll be able to access `storage` with this `pub`

Reply:
Oops, accidentally left that, not needed.  Fixed now.

Reply:
The function ndiffs did not need to be moved but I can do that if you think it's better.